packages/gasket-plugin-elastic-apm/CHANGELOG.md

```diff
@@ -1,18 +1,21 @@
 # `@gasket/plugin-elastic-apm`
 
-### 6.27.1
+- Add a `transactionName` lifecycle
+- Add a `transactionLabels` lifecycle
+
+## 6.27.1
```

Comment:
@agerard-godaddy do you recall why all of our CHANGELOG.md files jump from # to ###? I feel like there was a specific reason, but I can't remember what it was.

Reply:
I don't recall the reason, but it has been the pattern so yea if you would, leave these at `###` for now. If we want them at `##`, then let's do a holistic change across all package changelogs so that we can remain consistent.